Frontend/src/hooks/useUserData.ts

```diff
@@ -0,0 +1,15 @@
+import { useQuery } from '@tanstack/react-query'
+import { getUsersInfo } from '../api/user/post/get_user_info'
+
+export function useUserData(ids: number[]) {
+  const sortedIds = ids.sort((a, b) => a - b)
```

Comment:
Why not just `ids.toSorted()`? Why do you even pass in a custom comparator?

Reply:
toSorted() does not exist. I added the compare function because the typescript linter wants it

Reply:
Huh??
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/toSorted

Reply:
That function is part of ESNext.Array (from 2023), which is not enabled in our typescript compiler options. I could add it?

Reply:
Yup, although according to https://stackoverflow.com/a/77569768 the newer TypeScript releases (5.2 and after) support it natively.

I couldn't find the place where we pin our TS version, as it doesn't seem to be in package.json. Can you shed some light on this?

Reply:
I never explicitly added it because we only use it for the library functions. It's just a transitory dependency from openapi-typescript and set to "version": "5.1.6". I'll add it explicitly